README.md

```diff
@@ -44,5 +71,3 @@ dependencies {
     compileOnly "net.sacredlabyrinth.phaed.simpleclans:SimpleClans:2.12.2"
 }
 ```
-
-Some examples on how to use the API: https://github.com/RoinujNosde/SimpleClans/wiki/SimpleClans-API-example
```

Comment:
I think it's best to leave this comment, as it's relevant to the Developers section.
(only the link needs fixing)

Reply:
The link was moved to the top with the other navigation.

Reply:
I saw that, but having it close to the maven stuff will be of more help as it's usually the first thing I look for when checking another repo.

Reply:
Re-added with title.

Reply:
Thanks